packages/loot-core/src/server/accounts/parse-file.ts

```diff
@@ -6,8 +6,17 @@ import { looselyParseAmount } from '../../shared/util';
 
 import qif2json from './qif2json';
 
-export function parseFile(filepath, options?: unknown) {
-  let errors = [];
+type ParseError = { message: string; internal: string };
+export type ParseFileResult = {
+  errors?: ParseError[];
+  transactions?: unknown[];
```

Comment:
(related to the other comment)

Can we instead change this type to be specific? Then we could get rid of the `any`.

Reply:
As I said above — it truly is `unknown` — e.g. for CSV files the objects are keyed by whatever columns the user decided to enter and the mapping is currently handled on the client side.

Reply:
So if it is really `unknown` - doesn't it point to a bigger issue here that we're hiding by casting it to `any`? Could we add a type check there?

```
transactions.map(trans => {
  if (isRealObject(trans) {
    return { ...trans, otherAttr: trans.other }
  }
  throw new Error('Some issue');
});
```

Reply:
Yes, but restructuring transaction import is on my short list of tasks so I don’t want to put too much effort into something that will be replaced soon.

Reply:
Ok, that's pragmatic. Thanks for bearing with me!